checks/checkforfile.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package checks
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"io"
+	"strings"
+
+	"github.com/ossf/scorecard/checker"
+)
+
+// CheckIfFileExists downloads the tar of the repository and calls the predicate to check
+// for the occurence.
+func CheckIfFileExists(c checker.Checker, predicate func(name string,
+	Logf func(s string, f ...interface{})) bool) checker.CheckResult {
+	const pass int = 10
+	passResult := checker.CheckResult{
```

Comment:
can we move this and const pass int=10 in checker/check.go, next to InconclusiveResult. PassResult would be a common use result.